packages/editor/src/utils/l10n.js

```diff
@@ -0,0 +1,9 @@
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+export const l10n = {
```

Comment:
I'm guessing the goal is to have a central place for these localized things. 
I feel like "editor" is not the right package for that. Maybe a dedicated package like `@wordpress/locale` or something. 

Also, since it's only used in one place now, maybe it's fine to start adhoc.

Reply:
I think this is probably aiming to implement a function equivalent to WP Core's `wp_get_list_item_separator()` function, as mentioned in [this comment](https://github.com/WordPress/gutenberg/issues/58201#issuecomment-1908456742). Personally, I think adhoc is fine for now.

Reply:
> Can you share more about this? Why is this in the editor package? Can we instead just use the string where we need it?

@youknowriad I tried to explain that in the PR summary:

> Introduces a l10n object in the editor/utils package. For now, it only contains a listItemSeparator string. In the future this could be expanded with more strings, reshaped, or even moved to a separate package if desired. Pleaae let me know if the current approach is acceptable 

I don't think we should fix it on a 'per case' basis as there are other occurrences. I created https://github.com/WordPress/gutenberg/issues/58265

Glad to move to a new package if desired. I think in the future there would be the need for _more_ l10n things but for now it's just this tiny string...

Reply:
For this particular case,  I think it will make the code simpler if we use just the string `.tff, .otf, .woff, .woff2`. The loop to iterate over the file extensions is not necessary and it seems like an overkill because that info is unlikely for a good amount of time.